Site/Site.php

```diff
@@ -236,6 +236,7 @@ public static function getConfigDefinitions()
 			'analytics.enabled'                          => true,
 			'analytics.google_account'                   => null,
 			'analytics.google4_account'                  => null,
+			'analytics.google4_account_aoa'              => null,
```

Comment:
Is there a way we can add this without a specific reference to `aoa` in the Site package? Making the analitics account IDs instance away?

Reply:
Could the `InstanceConfigSetting` table help us here? 

https://github.com/silverorange/site/blob/3f857022c0122abba70e9cd24e885d01433916cb/Site/SiteConfigModule.php#L64-L92

Reply:
Ok yes, InstanceConfigSetting is the way to go. No need to change site package or even the config ini.

SQL insert will be something like:

`insert into InstanceConfigSetting (instance, is_default, name, value) values (43, false, 'analytics.google4_account', 'G-XXXXXXXX')`